src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json

```diff
@@ -3650,8 +3860,28 @@
       }
     }
   },
-  "slider-track-thickness": {
-    "value": "2px",
+  "slider-track-height-medium": {
+    "value": "4px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "slider-track-height-medium"
+      }
+    }
+  },
+  "slider-track-height-large": {
+    "value": "16px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "slider-track-height-large"
+      }
+    }
+  },
+  "🚫 slider-track-thickness": {
+    "value": "{slider-track-height-medium}",
```

Comment:
@GarthDB @PaliwalSparsh @lynnhao @mrcjhicks 
I don't like to waste everybody time on this, but I cannot approve this change without all of you looking at it and tell me it is okay. 

What I see here, in this line (and above) is:
`slider-track-thickness` value was `2px`
`slider-track-thickness` is deprecate, now.
`slider-track-thickness` value point to `slider-track-height-medium`, now.
`slider-track-thickness` value is `4px`, now.

Users, that have been using the `slider-track-thickness` token before this change, will get a new value from the `slider-track-thickness` token, when using it, if we merge this.

Deprecation does not include a value change. And it should not. 

please change the value of "🚫 slider-track-thickness" back to "2px".

Reply:
Thankyou for the feedback and discussion on this topic. I have changed the value of "🚫 slider-track-thickness" back to "2px".